Anytype/Sources/PresentationLayer/Modules/AllContent/AllContentSavedStates/AllContentSavedStatesService.swift

```diff
@@ -0,0 +1,28 @@
+import AnytypeCore
+
+protocol AllContentSavedStatesServiceProtocol: AnyObject {
```

Comment:
Consider AllContent~Saved~State~s~**Storage**Protocol

Reply:
Sounds good, will be renamed in the next PR